src/components/Board/Board.tsx

```diff
@@ -0,0 +1,63 @@
+import { DragDropContext, Droppable } from "react-beautiful-dnd";
+import { useDispatch, useSelector } from "react-redux";
+import { createList, dropCard } from "../../redux/reducers/listReducer";
+import { listSelector } from "../../redux/reducers/selectors";
+import { List } from "../List/List";
+import { BoardHeader } from "./BoardHeader";
+import * as styles from "./styles.css";
+
+type Props = {
```

Comment:
It's better to use Interfaces for object type desictition 
Also there is a naming patter ComponentNameProps
```suggestion
interface BoaradProps {
```
Same for other components